docs/guides/qiskit-add-ons.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Qiskit add-ons
+description: Understand the Qiskit add-on tools , which help you build utility-grade quantum workflows.
+---
+
+# Qiskit add-ons
+
+Qiskit add-ons are a collection of modular tools for building utility-scale workloads powered by Qiskit. These tools are derived from advanced research capabilities and are designed to be stackable with other tools as part of a [Qiskit pattern.](./intro-to-patterns)
+
+## Mapping tools
+
+### Multi-product formulas
+
+Multi-product formulas (MPF) reduce the Trotter error of Hamiltonian dynamics through a weighted combination of several circuit executions.
+
+- Visit the [GitHub repository.](https://github.com/Qiskit/qiskit-addon-mpf)
+- Read the [documentation.](https://qiskit.github.io/qiskit-addon-mpf)
+
+## Optimizing tools
+
+### Operator backpropagation
+
+Operator backpropagation (OBP) reduces circuit depth by trimming operations from the end at the cost of more operator measurements.
+
+- Visit the [GitHub repository.](https://github.com/Qiskit/qiskit-addon-obp)
+- Read the [documentation.](https://qiskit.github.io/qiskit-addon-obp)
+
+### Circuit cutting
+
+Circuit cutting reduces the depth of transpiled circuits by decomposing long range gates.
+
+- Visit the [GitHub repository.](https://github.com/Qiskit/qiskit-addon-cutting)
+- Read the [documentation.](https://qiskit.github.io/qiskit-addon-cutting)
+
+## Postprocessing tools
+
+### Sample-based quantum diagonalization
+
+Sample-based quantum diagonalization (SQD) classically postprocesses noisy quantum samples to yield more accurate energy estimations.
```

Comment:
"... to yield more accurate energy estimations of quantum system Hamiltonians, for example in chemistry applications."